src/alfasim_sdk/alfasim_sdk_api/api.h

```diff
@@ -778,4 +778,38 @@ DLL_EXPORT int get_liq_liq_shear_force_per_volume_input_variable(
     void* ctx, double* out, const char* var_name, int phase_id
 );
 
+/*!
+    Gets the relative emulsion viscosity for liquid-liquid calculations.
+
+    During the implementation of any HOOK related to the Liquid-Liquid Mechanistic model, this
+    function provides the relative emulsion viscosity from Emulsion Model (selected through GUI).
+
+    It allows the plugin HOOKs to use the same emulsion model used internally by ALFAsim.
+
+    The definition of relative viscosity is given by the ratio between the apparent viscosity
+    (dispersed field + continuous field) and the continuous field.
+
+    @param[in] ctx ALFAsim's plugins context.
+    @param[out] out Relative Emulsion Viscosity [-].
+    @param[in] mu_disp Dispersed Field Viscosity [Pa.s].
+    @param[in] mu_cont Continuous Field Viscosity [Pa.s].
+    @param[in] alpha_disp_in_layer   Dispersed Field Volume Fraction in the layer (emulsion) [m3 of dispersed field /m3 of layer].
```

Comment:
```suggestion
    @param[in] alpha_disp_in_layer Dispersed Field Volume Fraction in the layer (emulsion) [m3 of dispersed field /m3 of layer].
```

I'm not sure how the indentation works here, but the spacing is different, is it supposed to be like that?